EvalData/models/pairwise_assessment_document.py

```diff
@@ -1090,3 +1091,1067 @@ def completed_results_for_user_and_campaign(cls, user, campaign):
         ).values_list('item_id', flat=True)
 
         return len(set(results))
+    
+    
+@AnnotationTaskRegistry.register
+class PairwiseAssessmentDocumentESATask(BaseMetadata):
+    """
+    Models a pairwise assessment document evaluation task.
+
+    Note: this task is, similarily to other models, a shameless copy of

```

Comment:
Please update or delete this part of the comment. I think we can just mention which class(es) it's been copied from.

Reply:
Changed.